src/components/shared/TreeView.css

```diff
@@ -177,10 +177,16 @@
   opacity: 0.7;
 }
 
+/* stylelint-disable order/properties-order, declaration-block-no-duplicate-properties, value-keyword-case */
 .treeViewBody:focus .treeViewRow.isSelected {
-  background-color: highlight;
-  color: highlighttext;
-}
+  background-color: Highlight;
+  color: HighlightText;
+  background-color: -apple-system-selected-content-background;
+  color: -apple-system-alternate-selected-text;
```

Comment:
It would be nice to know when Safari will implement this. Do you know a webkit bug about that (or maybe file one) and add a link to it in a comment here?

Reply:
Good point. I've filed https://bugs.webkit.org/show_bug.cgi?id=230007 and will add a comment.